extension/src/prototypes/ui-components/ColorSetListItem.tsx

```diff
@@ -66,12 +95,41 @@ export const ColorSetListItem: FC<ColorSetListItemProps> = ({
     [setColor],
   );
 
+  const handleStrokeChange: ColorChangeHandler = useCallback(
+    result => {
+      setIndeterminateStrokeColor(result.hex);
+      if (continuous) {
+        setColor(color => ({
+          ...color,
+          strokeColor: result.hex,
+        }));
+        onChangeRef.current?.();
+      }
+    },
+    [continuous, setColor],
+  );
+
+  const handleStrokeChangeComplete: ColorChangeHandler = useCallback(
+    result => {
+      setColor(color => ({
+        ...color,
+        strokeColor: result.hex,
+      }));
+      onChangeRef.current?.();
+    },
+    [setColor],
+  );
+
   return (
     <>
       <ListItem disableGutters>
         <Stack direction="row" spacing={0.5} alignItems="center">
-          <StyledIconButton {...bindTrigger(popupState)}>
-            <ColorIcon color={indeterminateColor} />
+          <StyledIconButton>
+            <ColorIcon color={hasStroke ? indeterminateStrokeColor : indeterminateColor} />
+            <MainTrigger {...bindTrigger(hasStroke ? strokePopupState : popupState)} />
```

Comment:
This button UX become complex since it needs to behavior differently when click on center and click on outer part. Currently MUI throws warning like should not bind click event on children of IconButton but i've no other good solution for now. 
We can refactor this in the future if needed.